ci_scripts/build_documentation.rb

```diff
@@ -112,188 +133,122 @@ def copy_readme_and_fix_relative_links(readme_file, github_file_prefix, github_r
-      readme_args = "--readme '#{readme_temp_file}'"
+      readme_temp_file_generated = copy_readme_and_fix_relative_links(File.expand_path(readme, "#{$SCRIPT_DIR}/..").to_s, github_file_prefix, github_raw_file_prefix, m['framework_name'])
+      readme_temp_file = "#{docc_container}/#{m['framework_name']}.md"
+      `mkdir -p #{docc_container}`
+      `mv '#{readme_temp_file_generated}' '#{readme_temp_file}'`
     end
 
-    info "Executing jazzy for #{m['framework_name']}..."
-    `jazzy \
-      --config "#{$JAZZY_CONFIG_FILE}" \
-      --output "#{output}" \
-      #{readme_args} \
-      --github-file-prefix "#{github_file_prefix}" \
-      --title "#{docs_title(release_version)}" \
-      --module #{m['framework_name']} \
-      --sourcekitten-sourcefile "#{sourcekitten_files.join(',')}" \
-      --xcodebuild-arguments -destination,'generic/platform=iOS'`
+    info "Executing xcodebuild for #{m['framework_name']}..."
+
+    # Regenerate the project after adding the docc bundle
+    `tuist generate -n`
+
+    # Build the docs
+    puts `xcodebuild docbuild \
+     -scheme #{m['scheme']} \
+     -destination "generic/platform=iOS" \
+     -sdk iphoneos \
+     -configuration Release \
+     -derivedDataPath "#{$TEMP_BUILD_DIR}" \
+     OTHER_DOCC_FLAGS="--transform-for-static-hosting --hosting-base-path #{$HOSTING_BASE_PATH}#{m['framework_name'].downcase}/"`
 
-    # Delete temp readme file
-    File.delete(readme_temp_file) unless readme_temp_file.nil? || !File.exist?(readme_temp_file)
+    # Delete temp readme docc
+    FileUtils.remove_entry_secure(docc_container) if File.exist?(docc_container)
 
-    # Verify jazzy exit code
-    jazzy_exit_code = $?.exitstatus
+    # Verify exit code
+    xcodebuild_exit_code = $?.exitstatus
 
-    die "Executing jazzy failed with status code: #{jazzy_exit_code}" if jazzy_exit_code != 0
+    die "Executing xcodebuild failed with status code: #{jazzy_exit_code}" if jazzy_exit_code != 0
+    `mv #{$TEMP_BUILD_DIR}/Build/Products/Release-iphoneos/#{m['framework_name']}.doccarchive #{docs_root_directory}/docs/#{m['framework_name'].downcase}`
+    # Clean up the bogus index.html file created by DocC
+    File.delete("#{docs_root_directory}/docs/#{m['framework_name'].downcase}/index.html")
   end
 end
 
-# Creates html that lists all modules with docs enabled and links to their docs
+# Creates an index page that lists all modules with docs enabled and links to their docs
 # directory. The descriptions of the modules are from their podspec summaries.
-def index_page_content(modules)
-  view = Mustache.new
-  view.template_file = File.join_if_safe($ROOT_DIR, $JAZZY_CONFIG['theme'], 'templates', 'index.mustache')
+def build_index_page(modules, release_version, docs_root_directory)
+  # Copy the docc bundle template
+  temp_docc_dir = Dir.mktmpdir('stripe-docs-index-build')
+  `rsync -av #{$SCRIPT_DIR}/docs/Stripe.docc #{temp_docc_dir}/`
```

Comment:
I actually did use quotes here in an earlier version. But wildcard globbing doesn't work in quotes, and the later rsync invocations do use globbing, so I updated these earlier invocations for consistency. I probably should be using native Ruby here instead of globbing with rsync tbh.

Reply:
I think if this works, we can keep it for now.. not too concerned about switching over to native ruby unless this poses some sort of security threat -- which I don't think it does.

Reply:
Never mind, looked up some Bash guides and I just have to move the wildcard outside the quotes.